app/scripts/views/mixins/signup-mixin.js

```diff
@@ -0,0 +1,64 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// Shared implementation of `signUp` view method
+
+define(function (require, exports, module) {
+  'use strict';
+
+  module.exports = {
+    /**
+     * Sign up a user
+     *
+     * @param {object} account
+     * @param {string} password
+     * @return {object} promise
+     */
+    signUp: function (account, password) {
+      var self = this;
+      return self.invokeBrokerMethod('beforeSignIn', account.get('email'))
+        .then(function () {
+          return self.user.signUpAccount(account, password, self.relier, {
+            resume: self.getStringifiedResumeToken()
+          });
+        })
+        .then(self.invokeBrokerMethod.bind(self, 'afterSignUp'))
+        .then(self.onSignUpSuccess.bind(self, account));
```

Comment:
Document why the account is part of the call to `bind` here, whereas in the signin-mixin it is not.